pontoon/base/static/js/themeSwitcher.js

```diff
@@ -0,0 +1,77 @@
+$(function () {
+  function getSystemTheme() {
+    if (
+      window.matchMedia &&
+      window.matchMedia('(prefers-color-scheme: dark)').matches
+    ) {
+      return 'dark';
+    } else {
+      return 'light';
```

Comment:
We should default to dark, not light. So if the user sets "System" in Pontoon and "Auto" on macOS, she should get the dark theme. Right now she gets the light theme.